java/rocksjni/abstract_associative_merge_operator.cc

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
I think these need the Facebook copyright, although maybe that's auto-applied during upstream merge.

Reply:
i can forgive this patch to facebook. It is possible to add fb copyright